### ISSUE TITLE
Please define hg39 (or correct typo?)

### ISSUE THREAD

**Issue by AngieHinrichs** [EfresBR/G4iMGrinder#4]
Results.rar contains two files that include "hg39" in the name: hg39PQS.RData and hg39PiMS.RData.  What is hg39?  

hg19 is UCSC's database for the GRCh37 assembly, hg38 for GRCh38... but there is no GRCh39, or rather its release has been "indefinitely postponed" according to https://www.ncbi.nlm.nih.gov/grc .  I think using the name "hg39" without defining it and/or linking to the assembly that you used is going to cause a lot of confusion for those accustomed to hg19/hg38.  

I guess it's too late to define hg39 in your NARGaB publication (congrats on that!), but I hope you can at least add a clarification (and perhaps download files with a more recognizable name?) to github.

P.S. asking on behalf of a UCSC Genome Browser user who asked the Genome Browser team how to get hg39...

**Comment by EfresBR** [comment]
Hello,

Thanks for pinpointing the issue.
It seems I mistyped the name of the archives when saving it in RData format. I will very shortly change it to make it clear that the genome used was hg38.

After checking the publication (thank you on your congratulations btw), this error is only located in the archive's names themselves.
On the supplementary material, (section 9 - Genomes used and results with all methods) the first line reads: The complete human chromosomic genome (GRCh38.p12, Genome Reference
Consortium Human Build 38, INSDC Assembly GCA_000001405.27– gh38) was downloaded
from the Sanger Institute (www.sanger.ac.uk) in May 2019. 

However I just realised that I, very inteligently, reTAGed the genome to gh38 instead of hg38. As there is not much I can do now with the supplementary material, I will include in the package readme result section the human genome ref used.  

Would this suffice?

Thanks again

Efres
